stan/math/prim/scal/prob/beta_binomial_lcdf.hpp

```diff
@@ -103,10 +103,9 @@ namespace stan {
         const T_partials_return mu = alpha_dbl + n_dbl + 1;
         const T_partials_return nu = beta_dbl + N_dbl - n_dbl - 1;
 
-        const T_partials_return F = F32((T_partials_return)1, mu,
-                                                    -N_dbl + n_dbl + 1,
-                                                    n_dbl + 2, 1 - nu,
-                                                    (T_partials_return)1);
+        T_partials_return F;
+        F = F32((T_partials_return)1, mu, -N_dbl + n_dbl + 1, n_dbl + 2,
+          1 - nu, (T_partials_return)1);
```

Comment:
This pattern is a hold-over from original code, I don't think this will make it into 2.15 if I have to figure out the template stuff here (I've never thought it through before).   Could we push this to a new issue?